examples/tools/scene_viewer.rs

```diff
@@ -318,10 +317,12 @@ fn update_lights(
         projection_adjustment.z += SCALE_STEP;
     }
     for (_, mut light) in &mut query {
-        light.shadow_projection.left *= projection_adjustment.x;
-        light.shadow_projection.right *= projection_adjustment.x;
-        light.shadow_projection.bottom *= projection_adjustment.y;
-        light.shadow_projection.top *= projection_adjustment.y;
+        if let ScalingMode::Fixed { width, height } = light.shadow_projection.scaling_mode {
```

Comment:
This is one case where adjusting l/r/b/t automatically with `ScalingMode::Fixed` makes modifying it a bit more difficult.